src/main/java/seedu/savenus/ui/InfoWindow.java

```diff
@@ -14,15 +14,18 @@
 import seedu.savenus.model.info.BudgetInfo;
```

Comment:
I changed the budget command so the period of time is not compulsory anymore, help to reflect that thanks :)